salt/states/file.py

```diff
@@ -278,6 +278,7 @@ def run():
 import traceback
 from collections import Iterable, Mapping, defaultdict
 from datetime import datetime   # python3 problem in the making?
+from glob import iglob
```

Comment:
Please just import `glob` here and change references to `glob.iglob()`. You don't anything in terms of performance or memory by importing a single function, and it also has the side-effect of getting picked up by Sphinx when we publish the docs.